packages/ndla-ui/src/Image/Image.tsx

```diff
@@ -7,9 +7,16 @@
  */
 
 import React, { ReactNode } from 'react';
+import { breakpoints, mq } from '@ndla/core';
 import styled from '@emotion/styled';
 import LazyLoadImage from './LazyLoadImage';
 
+const ExpandButtonWrapper = styled.div`
+  ${mq.range({ until: breakpoints.mobileWide })} {
+    display: none;
+  }
+`;
```

Comment:
Dette burde for så vidt fungere bra, men all tilhørende styling er per nå i component.figure.scss, så burde likevel gjøre stylingen der

Reply:
Ja stemmer. I fix